src/schema.js

```diff
@@ -5,38 +5,49 @@ const DEFAULT_LABEL = "…"
 const blocks = {
     api: {
         parents: ['server'],
+        description: "Bundles endpoints under a certain `base_path`.",
+        examples: ['api-proxy'],
         labels: [null, DEFAULT_LABEL]
     },
     backend: {
         parents: ['beta_oauth2', 'definitions', 'jwt', 'oauth2', 'oidc', 'proxy', 'request'],
+        description: "Defines the connection to a local/remote backend service.",
+        examples: ['backend-configuration'],
         labelled: (parentBlockName) => {
             return parentBlockName === "definitions"
         }
     },
     basic_auth: {
         parents: ['definitions'],
+        description: "Access control for Basic Authentication.",
         labelled: true
     },
     beta_oauth2: {
         parents: ['definitions'],
+        description: "Access control for an OAuth2 Authorization Code Grant Flow redirect endpoint.",
         labelled: true
     },
     cors: {
         parents: ['api', 'files', 'server', 'spa'],
+        description: "Configures CORS (Cross-Origin Resource Sharing) behavior.",
         labelled: false
     },
     defaults: {
+        description: "Sets default values.",
         labelled: false
     },
     definitions: {
+        description: "Defines configurations for reuse.",
         labelled: false
     },
     endpoint: {
         parents: ['api', 'server'],
+        description: "Defines the entry points of Couper.",
         labels: ['/']
     },
     error_handler: {
```

Comment:
The link to the documentation must be `ERRORS.md#error_handler-specification`.

Reply:
See above. The `error_handler` block is simply missing in the reference. Details on how error handling works belong into `ERRORS.md`, but the reference should provide an anchor for each configuration element.

Reply:
In https://github.com/avenga/couper/pull/464, I added a section about the `error_handler` block to REFERENCE.md.